lib/tasks/data.rake

```diff
@@ -1,9 +1,12 @@
 require 'rake'
 
-# require 'rest-client'
+begin
+  require 'dotenv/load'
+rescue LoadError
+  puts 'dotenv is not available in this environment'
```

Comment:
@diiq We need `dotenv` locally to run this task, but not on review apps (production). 

Is there a better way to do this?

Reply:
we could manually, 
1. manually set the environment variables rather than use dotenv in production. eek!
2. if heroku is setting the environment variables then we can kill .env file and remove the "require 'dotenv/load'" line.